### DISCUSSION TITLE
Problem with CTX - invalid syntax

### DISCUSSION THREAD

**Discussion by Su870** [aertslab/SCENIC#360] [* General SCENIC questions] [answered]
I have included my code below. But I keep getting a syntax error when I use ctx. Anyone know how I can fix it?

import numpy as np
import pandas as pd
from arboreto.utils import load_tf_names
from arboreto.algo import grnboost2
import scanpy as sc
import loompy as lp
import pylab as plt
import pyscenic
import seaborn as sns
import matplotlib.pyplot as plt
import glob
import discord
import ctx
import ctxcore

pyscenic ctx TF_networks.tsv \
            {f_db_names} \
            --annotations_fname {f_motif_path} \
            --expression_mtx_fname {f_loom_path_scenic} \
            --output reg.csv
            --mask_dropouts \
            --num_workers 20

pyscenic ctx TF_networks.tsv \
              ^^^ (these arrows point towards the ctx)
SyntaxError: invalid syntax

Thanks!

**Comment by GirayEryilmaz** [comment, marked as answer]
It looks like you are trying to use the command line tool from the python code.
